.circleci/config.yml.j2

```diff
@@ -693,11 +813,34 @@ commands:
               if DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect "<< parameters.prod_docker_repo_docker_hub >>:$tag" >/dev/null 2>&1; then
                 echo "Image with Tag ("<< parameters.prod_docker_repo_docker_hub >>:$tag") already exists"
                 export NEW_POINT_RELEASE=false
+
               elif [[ "$tag" == "<< parameters.tag >>-$CIRCLE_BUILD_NUM" ]] || <<parameters.dev_release >> ; then
                 echo "Tagging and pushing image <<parameters.image_name>>:$tag to << parameters.dev_docker_repo_quay_io >>"
                 docker tag "$image" "<< parameters.dev_docker_repo_quay_io >>:${tag}"
                 docker push "<< parameters.dev_docker_repo_quay_io >>:${tag}"
 
+                echo "Build TAG artifact using environment variables and into DB"
```

Comment:
```suggestion
                echo "Build TAG artifact using environment variables and add it to DB in the QA Environment"
```

Reply:
Resolved in ac7c5efc38ae522578750647ada2da4bb67b10e1.